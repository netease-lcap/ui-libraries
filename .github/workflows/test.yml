name: Test CI

on:
  pull_request:
    branches: [ "*", "release/*"]

jobs:
  test:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v3

    - name: Set node
      uses: actions/setup-node@v3
      with:
        node-version: '18'

    - name: Install pnpm
      run: npm install -g pnpm

    - name: Install Dependencies
      run: pnpm install

    - run: npm run test
