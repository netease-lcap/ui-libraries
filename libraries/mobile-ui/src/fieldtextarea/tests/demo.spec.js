import Demo from '../demos';
import { snapshotDemo } from '../../../test/demo';

snapshotDemo(Demo);
