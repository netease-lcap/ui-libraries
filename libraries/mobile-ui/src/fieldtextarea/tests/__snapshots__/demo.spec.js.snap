// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders demo correctly 1`] = `
<div>
  <div>
    <div class="van-fieldtextarea__newwrap">
      <div class="van-fieldtextarea__wrap-con"><span class="van-fieldtextarea__control van-fieldtextarea__control--custom van-fieldtextarea__control--min-height" style="height: auto;">我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入我是多行输入</span></div>
    </div>
  </div>
  <div>
    <div class="van-fieldtextarea__newwrap van-fieldtextarea__newwrap--limit" style="height: 120px; background-color: bisque;">
      <div class="van-fieldtextarea__wrap-con"><textarea role="fieldtextarea" placeholder="" class="van-fieldtextarea__control van-fieldtextarea__control--custom van-fieldtextarea__control--min-height" style="height: auto;"></textarea></div>
      <div class="van-fieldtextarea__word-limit"><span class="van-fieldtextarea__word-num">99</span>/100</div>
    </div>
  </div>
</div>
`;
